config_insal_spatial-dyn_rgcn_pytorch.yml

```diff
@@ -0,0 +1,59 @@
+runDescription: "DRB Inland Salinity RGCN prediction with spatial holdout - dynamic attributes"
+
+# Input files
+obs_file: "2_process/out/drb_SC_obs_PRMS.zarr"
+attrs_file: "2_process/out/drb_attrs_PRMS.zarr"
+dist_matrix_file: "1_fetch/out/drb_distance_matrix.npz"
+# File containing the attribute names to use within the attrs_file
+x_vars_file: "4_predict/out/dynamic_attrs.txt"
+
+#river-dl code directory
+code_dir: "../river-dl"
+
+# Attribute names within obs_file
+y_vars: ['mean_value']
+#Attributs to take the log of for model training.
+log_y_vars: False
+
+# output location
+out_dir: "4_predict/out/spatial/RGCN/dynamic_test1"
+
+#random seed for training False==No seed, otherwise specify the seed
+seed: 21023

```

Comment:
Is this a date? Seems like a suspiciously specific `seed`.

Reply:
Yes, it's a date